interfaces/devdoc/file_util_requirements.md

```diff
@@ -0,0 +1,38 @@
+# file_util_linux
+
+## Overview
+
+Linux implementation of the file operation functions.
+
+## Exposed API 
+
+```c
+// Most defines removed for space constraints
+#define INVALID_HANDLE_VALUE            NULL
+
+typedef void *HANDLE;
+typedef void *LPSECURITY_ATTRIBUTES;
+
+MOCKABLE_FUNCTION(, HANDLE, file_util_create_file, const char*, lpFileName, unsigned long, desired_access, unsigned long, share_mode, LPSECURITY_ATTRIBUTES, lp_security_attributes, unsigned long, creation_disposition, unsigned long, flags_and_attributes, HANDLE, hTemplateFile);
+MOCKABLE_FUNCTION(, bool, file_util_close_handle, HANDLE, handle_input);
+```
+
+### file_util_create_file
+
+```c
+MOCKABLE_FUNCTION(, HANDLE, file_util_create_file, const char*, full_file_name, unsigned long, desired_access, unsigned long, share_mode, LPSECURITY_ATTRIBUTES, lp_security_attributes, unsigned long, creation_disposition, unsigned long, flags_and_attributes, HANDLE, h_template_file);
+```
+//share_mode, lp_security_attributes, flags_and_attributes, and h_template_file parameters are ignored in Linux
```

Comment:
nit: Consider putting this below the other sentence without the "//" comment (because it's an MD doc so we don't need c-style comments.